jobs/pgbackup/spec

```diff
@@ -62,17 +62,28 @@ properties:
     default: "*"
 
   pgbackup.s3.bucket:
-    description: "Amazon S3 bucket to store backups in"
+    description: "S3 bucket to store backups in"
     default: "pgbackup"
   pgbackup.s3.path:
     description: "Path to prepend to backup filenames when storing in Amazon S3"
     default: ""
   pgbackup.s3.access_key_id:
     description: "Access Key ID for storing backups in Amazon S3"
-    default: ""
   pgbackup.s3.secret_access_key:
     description: "Secret Access key for storing backups in Amazon S3"
-    default: ""
   pgbackup.s3.endpoint:
-    description: "endpoint for backup/restore"
-    default: "s3.amazonaws.com"
```

Comment:
Please restore default value as to avoid breaking compatibility of existing deployments